infra/messaging/src/index.js

```diff
@@ -77,6 +89,14 @@ app.get('/accounts/:address', async (req, res) => {
 
   address = Web3.utils.toChecksumAddress(address)
 
+  // If it's a proxy, make sure we're checking with the owner account address
+  if (await isContract(address)) {
```

Comment:
Would it make sense to put this kind of util functionality somewhere in a class. I expect this kind of `isContract` / `isProxyAddress` check will be  performed on many places.

Reply:
hm, actually, I might be able to pull these from the utils in `@origin/graphql` since that's now a dependency.  I keep forgetting about those.  Will see about that change this morning.  Thanks

Reply:
Moved `isContract()` to graphql utils